classes/image_populate_mfgtool.bbclass

```diff
@@ -0,0 +1,142 @@
+# Allow generation of mfgtool bundle
+#
+# The class provides the infrastructure for MFGTOOL generation and is tied to images. To generate
+# the bundle, the task populate_mfgtool must be called. For example:
+#
+# ,----[ Running populate_mfgtool for core-image-minimal image ]
+# | $: bitbake core-image-minimal -c populate_mfgtool
+# `----
+#
+# The class behavior is controlled through the MFGTOOLSCRIPT variable. The MFGTOOLSCRIPT variable
+# itself specifies a space-separated list of the scripts to enable. Following the scripts, you can
```

Comment:
To improve usability, I suggest updating the variable name with `_` and `S`: `MFGTOOL_SCRIPTS`.

Reply:
Fixed.